exercises/concept/lucians-luscious-lasagna/.meta/Exemplar.hs

```diff
@@ -0,0 +1,13 @@
+module LuciansLusciousLasagna exposing (elapsedTimeInMinutes, expectedMinutesInOven, preparationTimeInMinutes)
```

Comment:
please do make this line valid Haskell code (as you can see by the syntax highlighting it currently isn't), because under the idea that exemplars are something students should strive toward, we should expect that they strive toward valid Haskell code

Reply:
Done. I was mixing Elm syntax here.